bundle/libraries/libraries.go

```diff
@@ -68,6 +68,10 @@ func findArtifactsAndMarkForUpload(ctx context.Context, lib *compute.Library, b
 		return err
 	}
 
+	if len(matches) == 0 && isLocalLibrary(lib) {
+		return fmt.Errorf("no library found for %s", libPath(lib))
+	}
```

Comment:
What is the rationale for this additional check?

Reply:
When working on bundle example, I mistyped the library path in the task definition and it failed silently which was confusing